### DISCUSSION TITLE
Hypermode Commerce Example.

### DISCUSSION THREAD

**Discussion by trace2798** [hypermodeinc/docs#71] [General]
Creating this discussion to ask  questions related to the [hyper-commerce repo](https://github.com/hypermodeinc/hyper-commerce)

**Comment by trace2798** [comment]
Wondering about the future implementation of that example. Over the last few days, I have tried to convert the CSV data into the following formats JSON, JS object, and JSON in a batch of any number. Should I make a pull request with the data in new formats? The main benefit will be then the user can insert the data without using Python.

**Reply by johnymontana** [reply]
Hey @trace2798  - sure, if you have it working with data in other formats we'd love to see it. Maybe if you have it working in a fork you could share it here and we could give some feedback?

On the plans for the future implementation of that project, I think as we launch ModusDB we'll want to migrate the HyperCommerce project to make use of ModusDB but I'm not sure if that's been discussed much yet. @jairad26 @StephDietz What do you think?

**Reply by jairad26** [reply]
hey @trace2798 yea, getting rid of the python would be great would love to see that PR! And to follow up on Will's comment yea, we'll be moving collections in general off of postgres databases and onto ModusDB (it's an in-process db, kind of like sqlite that also stores on disk, more info on that to come soon), so theres less dependencies in general.

**Reply by trace2798** [reply]
@jairad26 @johnymontana Thanks for the response. I will create a fork with implementation with updated data format and let you all know. I have this code from my [hackathon submission](https://github.com/trace2798/trivia-modus/blob/master/components/button-upsert.tsx) which I used to upload the movie data but then I did not run collections locally directly with the deployed version and since the data was small I just the normal upsert. Now I will try to implement both and update you all.
